.github/workflows/linux.yml

```diff
@@ -23,13 +29,17 @@ jobs:
     - name: Install Qt
       uses: jurplel/install-qt-action@v3
       with:
-        version: '5.15.2'
+        version: ${{ matrix.qtversion }}
         host: 'linux'
         install-deps: 'true'
 
     - name: Install dependencies
       run: sudo apt-get install extra-cmake-modules libxcb-xfixes0-dev xvfb
 
+    - name: Install Qt6 dependencies
```

Comment:
This one is going to be installed for Gt5 and Qt6, right?

Reply:
I was going to add an "if" here, but then decided that it doesn't hurt qt5 either, and eventually qt5 build would be removed anyway...

So went with just a comment instead